mustuse-loader.php

```diff
@@ -213,7 +213,7 @@ public function get_mu_plugins() {
 			require ABSPATH . 'wp-admin/includes/plugin.php';
 		}
 		// Get all plugins
-		$mu_plugins = get_plugins( self::$wpmu_plugin_dir );
+		$mu_plugins = apply_filters( 'mustuse_loader_mu_plugins', get_plugins( self::$wpmu_plugin_dir ) );
```

Comment:
@cfoellmann Please. I like to switch `must_use_loader.mu_plugins` in his name. This is my current format for hooks in plugin, text domain DOT hook-name. Thanks! Please add also a small text to the readme about his hook and his possibility. Thx.

Reply:
In this context we should rename the fil to must-use-loader, Consistency with class, file name, repository.

Reply:
You should also add your example to remove Mercator from #10, it is helpful.